index.html

```diff
@@ -108,6 +108,39 @@ <h2>A project for first-time contributions</h2>
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
 
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="zelalem">Zelalem Mekonnen</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="www.twitter.com" target="_blank">@CaptainAfrica99</a>
+          </p>
+          <p class="about">Oh what a life!</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://eloquentjavascript.net/" target="_blank" title="First Resource">
+                  Resource 1
+                </a>
+              </li>
+              <li>
+                <a href="https://linuxjourney.com/" target="_blank" title="Second Resource">
+                  Resource 2
+                </a>
+              </li>
+              <li>
+                <a href="https://i.am.ai/" target="_blank" title="Third resource">
+                  Resource 3
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+
```

Comment:
add `<!-- ________*Zelalem* Contributor card END ________  --> after the end of your card